src/internal.c

```diff
@@ -2071,6 +2071,12 @@ int InitSSL(WOLFSSL* ssl, WOLFSSL_CTX* ctx)
     ssl->sessionSecretCb  = NULL;
     ssl->sessionSecretCtx = NULL;
 #endif
+
+#ifdef HAVE_SESSION_TICKET
+    ssl->session.ticket = ssl->session.staticTicket;
+    ssl->session.isDynamic = 0;
```

Comment:
2077 and 2078 are redundant, the memset 0 at the top of the function handles default inits.